parameter/Element.cpp

```diff
@@ -39,6 +39,8 @@
 
 using std::string;
 
+const std::string& CElement::gDescriptionPropertyName = "Description";
```

Comment:
You are using a rather obscure feature of c++: binding a const reference to a rvalue extends it's lifetime to that of the reference.
Are you sure you want to do that ? I would rather remove the `&`.